provider/vpc/command.py

```diff
@@ -1,132 +1,172 @@
 from ipaddress import ip_network
 
 from provider.vpc.diagram import VpcDiagram
-
 from shared.command import CommandRunner, BaseCommand
-from shared.common import *
+from shared.common import (
+    BaseOptions,
+    ResourceDigest,
+    VPCE_REGEX,
+    SOURCE_IP_ADDRESS_REGEX,
+)
 from shared.diagram import NoDiagram, BaseDiagram
 
 
 class VpcOptions(BaseOptions):
     vpc_id: str
 
     def __new__(cls, session, region_name, vpc_id):
+        """
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Multi-line docstring summary should start at the first line (D212)](https://app.codacy.com/gh/Cloud-Architects/cloud-discovery/pullRequest?prid=5655786)